<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotLegendWidget</class>
 <widget class="QWidget" name="PlotLegendWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="HeaderLayout" stretch="0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="LegendCaption">
       <property name="text">
        <string>Legend</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EditLegendButton">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../resource/icons/icon_action_edit.png</normaloff>../../resource/icons/icon_action_edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="ExceptionsLegendLayoutCaption">
     <property name="text">
      <string>Exceptions:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
